cours/aws/aws-compute.fr.md

```diff
@@ -0,0 +1,55 @@
+# AWS Calcul
+
+## EC2
+
+### Qu'est-ce qu'Amazon EC2 ?
+
+ - Elastic Compute Cloud
+ - Offre une capacité de calcul évolutive dans le cloud Amazon Web Services (AWS)
+ - Environnements de calcul virtuels, appelés **instances** (équivalent du serveur ou de la machine virtuelle)
```

Comment:
Only two spaces before a `-` and if you want to increment add two more and that goes on and on.